filter/ArticleCrossrefXmlFilter.php

```diff
@@ -113,8 +113,8 @@ public function createJournalArticleNode($doc, $submission)
 
         // title
         $titlesNode = $doc->createElementNS($deployment->getNamespace(), 'titles');
-        $titlesNode->appendChild($node = $doc->createElementNS($deployment->getNamespace(), 'title', htmlspecialchars($publication->getLocalizedTitle($locale), ENT_COMPAT, 'UTF-8')));
-        if ($subtitle = $publication->getData('subtitle', $locale)) {
+        $titlesNode->appendChild($node = $doc->createElementNS($deployment->getNamespace(), 'title', htmlspecialchars($publication->getLocalizedTitle($locale, 'html'), ENT_COMPAT, 'UTF-8')));
```

Comment:
I think this will lead these fields to be double-escaped. A title containing markup should look like:
```
<title>E=mc<sup>2</sup></title>
```
...but the call to `htmlspecialchars` will cause it to look like...
```
<title>E=mc&lt;sup&gt;2&lt;/sup&gt;</title>
```

Reply:
@asmecher `htmlspecialchars` was there before my changes . Was that a mistake ? Also considering the following title 
```
<title>Albert Einstein great formula "E=MC<sup>2</sup>" allow us to measure the released energy by nuclear fusion</title>
```
Do we allow `quotes(single or double)` in title ?

Reply:
`htmlspecialchars` made sense with plaintext content (it will make sure that `<` `>` `&` will be interpreted as text characters, not HTML entities) but now that we're allowing HTML it'll need to be changed for the appropriate filtering function. We do allow quotes in titles.